components/ModSubmissionsEditFormFacility.vue

```diff
@@ -286,20 +286,20 @@ const validateFields = () => {
     const isLongitudeValid: boolean = validateFloat(mapLongitude.value)
 
     return (
-        !isNameEnValid ||
```

Comment:
the non-config file changes are just from running `yarn lint:fix` which changed as part of the package upgrades and a fix I had to do to eslint config